README.md

```diff
@@ -209,8 +209,15 @@ authenticator.use(
       secret: process.env.ENCRYPTION_SECRET,
 
       createTOTP: async (data, expiresAt) => {
-        // Create the TOTP data in the database along with expiresAt.
-        await db.totp.create({ data, expiresAt })
+        await prisma.totp.create({ data: { ...data, expiresAt } })
+
+        try {
+          // Delete expired TOTP records.
+          // Better if this were in scheduled task.
+          await prisma.totp.deleteMany({ where: { expiresAt: { lt: new Date() } } })
```

Comment:
Uh, I'm not sure about this. If a database contains a large number of expired TOTPs, the creation and storage of the TOTP could take more time than necessary. Also, if that process fails, the authentication will fail, or am I mistaken about this?

Reply:
For someone who knows what they are doing and are building an application to scale. They would never do this and would put it in a scheduled task or some other mechanism outside of the `createTOTP`.

For the beginner or someone who is in a hurry just to get something working and doesn't need to scale, this gets them going without filling up their database with expired TOTP's. With an index on `expiresAt`, this should not be an expensive operation especially at small scale. 

The try/catch around it should prevent any delete errors from causing `createTOTP` to fail. Removing the `await` is very interesting idea.

```ts
  prisma.totp.deleteMany({where: { expiresAt: { lt: new Date() } } }).catch((reason) => console.warning(reason))
``` 

The code starts to look a little more obscure to the beginner who may be more comfortable with `await`.

Based on your feedback, I think we should remove `expiresAt` from the README code and have an `expiresAt` section in Customization. I can make that change.

Reply:
Yes, I think you are absolutely right. For the beginner or someone in a hurry, this is just fine, and it also gives the idea that it could be handled differently later at some point.

Nothing to change here, everything is just as good as it can be, @mw10013, and you thought about it very well and carefully!

Also, let's keep the `await` call in there, it's the most common way to call async functions and removing it could be somewhat a visual disruption for folks in a hurry to learn the basics of the Strategy.

Alright, with everything looking this good, I will take the time to create the release on GitHub.

As I said before, a million thanks for the amazing work you've been doing, testing, etc.
Merry Christmas to you!